src/platforms/ios/PingTest.swift

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import NetworkExtension
+
+class PingTest {
```

Comment:
🤩 NICE, even if we do not have automation, could you add a small doc on how to run those type of things on the machine? (if it is runnable ofc :D )

Reply:
This was poorly named. I renamed it to `PingAnalyzer`. It isn't a test, it's the helper function that sends test pings out.